### ISSUE TITLE
plot function not working 

### ISSUE THREAD

**Issue by vanzantom** [FreeFem/FreeFem-sources#108]
Dear all,

I recently upgraded FreeFem++ from 3.47 to 4.2.1.
Since the upgrade, the plot function of FreeFem++ does not work anymore, in the sense that FreeFem++ does not open any window plot.
The codes always run correctly, without giving any errors. However the plot calls (even with wait=1) do not create any windows to show the results.

Does everyone have a suggestion how to fix this problem?
I already removed and reinstalled from scratch the latest version.

Thank you a lot,

Tommaso

**Comment by sgarnotel** [comment]
Dear Tommaso,

Can you specify your OS ?

Simon

**Comment by vanzantom** [comment]
Dear Simon,

thank you for your reply.
I am using Ubuntu 18.04.03 .  After my post, I realised that there is a similar issue discussed in the community( sorry for posting here and not there..)
https://community.freefem.org/t/compiles-fine-but-no-graphics-in-freefem-executable-v4/42/3
which however was not very helpful in my case.

I used the binary installation. If it is helpul, the command which gives 
tommaso@tommaso:~$ which ffglut
/usr/bin/ffglut

Thanks,
Tommaso
